squid/templates/deployment.yaml

```diff
@@ -83,6 +108,8 @@ spec:
       - name: {{ template "squid.fullname" . }}-conf
         configMap:
           name: {{ template "squid.fullname" . }}-conf
+      - emptyDir: {}
```

Comment:
Could we setup this as a memory volume (to avoid disk writes) ? Or in function of the cache size, this is not wanted ? @bclermont

Reply:
In my case, I want to lower latency to retrieve files from outside the WAN.
and few instances of those squid will likely have few Tb storages.

So, cache in memory is not efficient.

Plus, if the goal is to have in-memory only cache for squid, there is something squid does already. without a filesystem <-> ramdisk kind of hack.

`emptyDir` is good for users who  need disk storage. memory storage need to use https://wiki.squid-cache.org/SquidFaq/SquidMemory

this PR only does something if there is a directive in squid configuration to do on-disk caching

Reply:
Thanks for the explanation, make sense!